src/concourse/pipelines/infrastructure/ovs/static_assets_pipeline.py

```diff
@@ -0,0 +1,171 @@
+        plan=[
+            GetStep(get=ovs_release.name, trigger=True),
+            TaskStep(
+                task=Identifier("build-ovs-staticfiles"),
+                privileged=True,
+                config=TaskConfig(
+                    platform=Platform.linux,
+                    image_resource=dcind_resource,
+                    inputs=[Input(name=ovs_release.name)],
+                    outputs=[Output(name="staticfiles"), Output(name="static")],
+                    # https://github.com/mitodl/odl-video-service/blob/master/.github/workflows/ci.yml#L63-L97
+                    params={
+                        "DEBUG": "False",
+                        "NODE_ENV": "production",
+                        "DATABASE_URL": "postgres://postgres:postgres@localhost:5432/postgres",  # pragma: allowlist secret
+                        "ODL_VIDEO_SECURE_SSL_REDIRECT": "False",
+                        "ODL_VIDEO_DB_DISABLE_SSL": "True",
+                        "CELERY_TASK_ALWAYS_EAGER": "True",
+                        "REDIS_URL": "redis://localhost:6379/4",
+                        "XDG_CACHE_HOME": "/src/.cache",
+                        "SECRET_KEY": "actions_secret",  # pragma: allowlist secret
+                        "AWS_ACCESS_KEY_ID": "fakeawskey",  # pragma: allowlist secret
+                        "AWS_SECRET_ACCESS_KEY": "fakeawssecret",  # pragma: allowlist secret
+                        "AWS_REGION": "us-east-1",
+                        "CLOUDFRONT_KEY_ID": "cfkeyid",
+                        "DROPBOX_KEY": "foo_dropbox_key",  # pragma: allowlist secret
+                        "ET_PIPELINE_ID": "foo_et_pipeline_id",
+                        "FIELD_ENCRYPTION_KEY": "jtma0CL1QMRLaJgjXNlJh3mtPNcgok0G5ajRCMZ_XNI=",  # pragma: allowlist secret
+                        "GA_DIMENSION_CAMERA": "fake",
+                        "GA_KEYFILE_JSON": "fake",
+                        "GA_VIEW_ID": "fake",
+                        "GA_TRACKING_ID": "fake",
+                        "LECTURE_CAPTURE_USER": "admin",
+                        "MAILGUN_URL": "http://fake_mailgun_url.com",
+                        "MAILGUN_KEY": "foookey",  # pragma: allowlist secret
+                        "ODL_VIDEO_BASE_URL": "http://video.example.com",
+                        "VIDEO_S3_BUCKET": "video-s3",
+                        "VIDEO_S3_TRANSCODE_BUCKET": "video-s3-transcodes",
+                        "VIDEO_S3_THUMBNAIL_BUCKET": "video-s3-thumbs",
+                        "VIDEO_S3_SUBTITLE_BUCKET": "video-s3-subtitles",
+                        "VIDEO_S3_WATCH_BUCKET": "video-s3-watch",
+                        "VIDEO_CLOUDFRONT_DIST": "video-cf",
+                        "YT_ACCESS_TOKEN": "fake",
+                        "YT_REFRESH_TOKEN": "fake",
+                        "YT_CLIENT_ID": "fake",
+                        "YT_CLIENT_SECRET": "false",  # pragma: allowlist secret
+                        "YT_PROJECT_ID": "fake",
```

Comment:
Are we able to trim/simplify this at all for the asset compilation step? /cc @mbertrand